README.md

```diff
@@ -105,4 +95,8 @@ If you have any questions or feedback regarding the Grafana Agent:
   [Discussions page](https://github.com/grafana/agent/discussions).
 * [File an issue](https://github.com/grafana/agent/issues/new) for bugs, issues
   and feature suggestions.
+  
+## Contributing
+
+Any contributions are welcome and details can be found [here](./cmd/agent/DEVELOPERS.md).
```

Comment:
nit: "in our developers guide" instead of "here" so people know where they're going